http/file_server.ts

```diff
@@ -255,7 +253,6 @@ export async function serveFile(
   return createCommonResponse(Status.OK, file.readable, { headers });
 }
 
-// TODO(bartlomieju): simplify this after deno.stat and deno.readDir are fixed
```

Comment:
Note: This is an ancient comment by Kevin and this doesn't seem relevant anymore. https://github.com/denoland/deno_std/pull/15/files#diff-106086daad2313990285a9c61085eeb5995d479127db151e613f84ea709450c8R104